### ISSUE TITLE
A sentence in Basic Image Classification Tutorial is not clear

### ISSUE THREAD

**Issue by worldpeaceaspirer** [tensorflow/tensorflow#47553]
## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/tutorials/keras/classification#make_predictions

## Description of issue (what needs changing): 
The statement, **The model's linear outputs, [logits](https://developers.google.com/machine-learning/glossary#logits).** can't be understood.

More explanation can be added on why the Outputs are Linear.

**Comment by innat** [comment]
In that blog, the outputs of the model are of course linear. They define the model as 

```python
model = tf.keras.Sequential([
    tf.keras.layers.Flatten(input_shape=(28, 28)),
    tf.keras.layers.Dense(128, activation='relu'),
    tf.keras.layers.Dense(10) # < ---------------- Look here
])
```

Here you can see they didn't use any activation and by default, it's `None`. And that makes it a linear layer. So the output would be `logit`. And further, they add another layer which is `softmax` to get `probabilities` from the `logit`. 

```
probability_model = tf.keras.Sequential([model, tf.keras.layers.Softmax()])
```

So, `model` here is giving linear outputs (`logit`) but when we add the `softmax` layer then it will give `probabilities` from the `logit`.

**Comment by worldpeaceaspirer** [comment]
@innat,
Thank you for your response. I agree with your explanation but my point is that we can add the same explanation in the Tutorial so that it will be clear to Newbies. Also, the sentence formation of the statement, **`The model's linear outputs, logits`**, is not proper and is incomplete.

**Comment by KiranKhanna721** [comment]
It is a image classification problem that is why we are using Softmax()  activation function in
probability_model = tf.keras.Sequential([model, tf.keras.layers.Softmax()])

**Comment by researchofhemanth** [comment]
@worldpeaceaspirer logits are normally raw outputs of a classification model. A Normalization function like softmax is added for normalized probabilities in the prediction. And the logits hyperlink clearly explains what logits are and how we can use normalization functions based on the type of classification problem. Beginners normally have an idea of logits and the use of activation functions so I guess there is no need to add anything new to the tutorial.

**Comment by 8bitmp3** [comment]
Reviewed. Thanks everyone!
